execute/transport.go

```diff
@@ -68,6 +68,7 @@ func (t *consecutiveTransport) RetractTable(id DatasetID, key flux.GroupKey) err
 }
 
 func (t *consecutiveTransport) Process(id DatasetID, tbl flux.Table) error {
+	tbl.RefCount(1)
```

Comment:
Where is the companion decrement to the increment?

Reply:
It's in process message. It was already there previously, but the increment was in unrelated code so I thought it would be easier to track if the increment and decrement were in the same structure.

That's why I removed the increment in the dataset because it relied on an implementation detail.